themes/pom.xml

```diff
@@ -19,6 +19,7 @@
         <maven.build.cache.exclude.value.2>src/main/resources/theme/keycloak/common/resources/vendor</maven.build.cache.exclude.value.2>
         <maven.build.cache.exclude.value.3>src/main/resources/theme/keycloak.v2/welcome/node_modules</maven.build.cache.exclude.value.3>
         <maven.build.cache.exclude.value.4>src/main/resources/theme/keycloak.v2/welcome/resources/vendor</maven.build.cache.exclude.value.4>
+        <maven.build.cache.exclude.value.5>src/main/resources/theme/keycloak.v2/account/src/web_modules</maven.build.cache.exclude.value.5>
```

Comment:
AFAIK the `web_modules` directory was only ever used by the Account V2 theme, which has since been removed. Why is is being added here?

Reply:
Thank you - removed that part. I was confused by an old `web_modules` folder in my workspace which was apparently never cleared from it. After deleting it and doing a full run, it didn't re-appear. So I removed those lines from the `pom.xml`. Please re-review. Thanks!